query/src/main/java/org/infinispan/query/backend/IndexModificationStrategy.java

```diff
@@ -49,19 +49,6 @@ public boolean shouldModifyIndexes(FlagAffectedCommand command, InvocationContex
       }
    },
 
-   /**
-    * Only events generated by the current node will trigger indexing events (excepting those flagged with {@code
-    * Flag.SKIP_INDEXING}.
-    */
-   @Deprecated
```

Comment:
it was deprecated in ISPN 10. should be removed in ISPN 11, right?

Reply:
no. today is the day it dies. this is not public api. I deprecated it by mistake . should have removed it altogether in https://github.com/infinispan/infinispan/pull/7532